src/scripts/oauth2-key-generation.js

```
@@ -438,18 +548,18 @@ async function removeApplicationKey() {
-    const scopeContainer = document.getElementById('scopeContainer-' + devAppId);
+    const scopeContainer = document.getElementById('scopeContainer-' + devAppId + '-' + keyType);
     const scopeInput = document.getElementById('scope-' + devAppId);
 
     if (!(subscribedScopes)) {
         // In the regenerate token request, the scopes are fetched from the span tags
-        const scopeElements = document.querySelectorAll(`#scopeContainer-${devAppId} .span-tag`);
+        const scopeElements = document.querySelectorAll(`#scopeContainer-${devAppId}-${keyType} .span-tag`);
         subscribedScopes = Array.from(scopeElements).map(el => el.textContent.replace('×', '').trim());
         scopeContainer.setAttribute('data-scopes', JSON.stringify(subscribedScopes));
-        tokenBtn = document.getElementById('regenerateKeyBtn');
+        tokenBtn = document.getElementById('regenerateKeyBtn-' + keyType);
```

Biome (2.1.2)

[error] Can't assign tokenBtn because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)